doc-content/enterprise-only/openshift/externaldb-build-proc.adoc

```diff
@@ -11,61 +11,79 @@ Complete the steps in this build procedure to provide drivers for any of the fol
+*** `docker`, provided by the `python3-docker` or similar package
+*** `docker-squash`, provided by the `python3-docker-squash` or similar package
+*** `behave`, provided by the `python3-behave` or similar package
+*** `s2i`, provided by the `source-to-image` or similar package
 
 .Procedure
-. For IBM DB2, Oracle Database, or Sybase, provide the JDBC driver JAR file in a local directory or on an HTTP server. Within the local directory or HTTP server, the following paths are expected:
+. For IBM DB2, Oracle Database, or Sybase, provide the JDBC driver JAR file in a local directory.
+. Download the `{PRODUCT_FILE}-openshift-templates.zip` product deliverable file from the {PRODUCT_DOWNLOAD_LINK}[Software Downloads] page of the Red Hat Customer Portal. 
+. Unzip the file and, using the command line, change to the `templates/contrib/jdbc` directory of the unzipped file. This directory contains the source code for the custom build.
+. Run one of the following commands, depending on the database server type:
 +
-** For IBM DB2, `<local_path_or_url>/com/ibm/db2/jcc/db2jcc4/10.5/db2jcc4-10.5.jar`
-** For Oracle Database, `<local_path_or_url>/com/oracle/ojdbc7/12.1.0.1/ojdbc7-12.1.0.1.jar`
-** For Sybase, `<local_path_or_url>/com/sysbase/jconn4/16.0_PL05/jconn4-16.0_PL05.jar`
+** For Microsoft SQL Server:
 +
-Replace `<local_path_or_url>` with the path to the local directory or the URL for the HTTP server where the driver is provided.
+[subs="attributes,verbatim,macros"]
+----
+make build mssql
+----
 +
-. Download the `{PRODUCT_FILE}-openshift-templates.zip` product deliverable file from the {PRODUCT_DOWNLOAD_LINK}[Software Downloads] page of the Red Hat Customer Portal. 
-. Unzip the file and, using the command line, change to the `templates/contrib/jdbc` directory of the unzipped file. This directory contains the source code for the custom build.
-. Change to one of the following subdirectories:
+** For MariaDB:
 +
-** For Microsoft SQL Server, `mssql-driver-image`
-** For MariaDB, `mariadb-driver-image`
-** For IBM DB2, `db2-driver-image`
-** For Oracle Database, `oracle-driver-image`
-** For Sybase, `sybase-driver-image`
+[subs="attributes,verbatim,macros"]
+----
+make build mariadb
+----
 +
-. Enter one of the following commands:
+** For IBM DB2:
 +
---
-** For Microsoft SQL Server or MariaDB:
-
 [subs="attributes,verbatim,macros"]
 ----
-../build.sh
+make build db2
 ----
-
-** For IBM DB2, Oracle Database, or Sybase:
-
++
+** For Oracle Database:
++
 [subs="attributes,verbatim,macros"]
 ----
-../build.sh --artifact-repo=<local_path_or_url>
+make build oracle artifact=/tmp/ojdbc7.jar version=7.0
 ----
-
-Replace `<local_path_or_url>` with the path to the local directory or the URL for the HTTP server where the driver is provided. Example:
-
++
+In this command, replace `/tmp/ojdbc7.jar` with the pathname of the downloaded Oracle Database driver and `7.0` with the version of the driver.
```

Comment:
"path name" two words, per IBM. Comes up a few times. Search all instances of "pathname" and replace.